Headers/DebugServer2/Base.h

```diff
@@ -45,7 +41,7 @@ typedef SSIZE_T ssize_t;
 #define OS_WIN32
 #elif defined(__FreeBSD__)
 #define OS_FREEBSD
-#elif defined(__APPLE__) && defined(TARGET_OS_MAC)
```

Comment:
I added the TARGET_OS_MAC check to be sure that we will be on a specific architecture. Since mac/iphone/etc could have different implementation I was thinking it will be better to crash at the compil time if someone try to compile on an non compatible arch

```c
if __APPLE__
   if MAC

   if IOS

   else...
```